app/build.gradle.kts

```diff
@@ -0,0 +1,162 @@
+            isUniversalApk = true
+        }
+    }
+
+    lint {
+        checkReleaseBuilds = false
+        disable.add("JvmStaticProvidesInObjectDetector")
+        disable.add("FieldSiteTargetOnQualifierAnnotation")
+        disable.add("ModuleCompanionObjects")
+        disable.add("ModuleCompanionObjectsNotInModuleParent")
+    }
+
+    testOptions {
+        unitTests {
+            isReturnDefaultValues = true
+            isIncludeAndroidResources = true
+        }
+    }
+
+    buildToolsVersion = "30.0.3"
+
+
+    buildFeatures {
+        dataBinding = true
+        viewBinding = true
+    }
+}
+
+repositories {
+    maven(url = "https://jitpack.io")
+}
+
+dependencies {
+    implementation(Deps.Multidex)
+    implementation(Deps.Androidx.Lifecycle.Extensions)
+    implementation(Deps.AndroidChart)
+    implementation(Deps.RxJava.RxJava)
+    implementation(Deps.RxJava.RxAndroid)
+    implementation(Deps.GoogleMaterial)
+    implementation(Deps.Androidx.ConstraintLayout)
+    implementation(Deps.Androidx.Core.Core)
+    implementation(Deps.Androidx.Core.Ktx)
+    implementation(Deps.Androidx.Lifecycle.Runtime)
+    implementation(Deps.Androidx.Lifecycle.Viewmodel)
+    implementation(Deps.Kotlin)
+    implementation(Deps.Coroutines)
+    implementation(Deps.Androidx.Fragment)
+    implementation(Deps.Androidx.Lifecycle.Livedata)
+    implementation(Deps.Androidx.Navigation.Fragment)
+    implementation(Deps.Androidx.Navigation.UI)
+    implementation(Deps.Androidx.Navigation.DynamicFeatures)
+    implementation(Deps.Androidx.Preferences)
+    implementation(Deps.Androidx.Worker)
+    implementation(Deps.Androidx.RecyclerView)
+    implementation(Deps.Hilt.Hilt)
+    kapt(Deps.Hilt.Kapt)
+
+    testImplementation(Deps.Testing.Androidx.Core)
+    testImplementation(Deps.Testing.Androidx.Runner)
+    testImplementation(Deps.Testing.Androidx.Rules)
+    testImplementation(Deps.Testing.Androidx.JUnitExtension)
+    testImplementation(Deps.Testing.Androidx.ArchCore)
+    testImplementation(Deps.Testing.Mockito.Core)
+    testImplementation(Deps.Testing.Mockito.Kotlin)
+    testImplementation(Deps.Testing.Robolectric)
+    testImplementation(Deps.Testing.JUnit)
+    testImplementation(Deps.Testing.Hamcrest)
+    testImplementation(Deps.Testing.Coroutines)
+    testImplementation(Deps.Testing.Mockk)
+    androidTestImplementation(Deps.Testing.Androidx.Navigation)
+
+    implementation(files("src/main/libs/jsoup-1.13.1.jar"))
+}
+
+kapt {
+    correctErrorTypes = true
+}
+
+spotless {
+    ratchetFrom("origin/develop")
+
+    java {
+        target(fileTree("dir" to "src", "include" to "**/*.java"))
+
+        googleJavaFormat("1.15.0")
+        removeUnusedImports()
+        trimTrailingWhitespace()
+        endWithNewline()
+    }
+    kotlin {
+        target(fileTree("dir" to "src", "include" to "**/*.kt"))
+        ktlint(Versions.KtLint).editorConfigOverride(mapOf(
+                "disabled_rules" to "no-wildcard-imports",
```

Comment:
I would reconsider allowing this. Abusing wildcard imports has potential in planting mine fields that can explode in far future and entirely different context.